app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -9,6 +9,8 @@ import FailedCaptureAttemptsContext from '../context/failed-capture-attempts';
 import DocumentCaptureWarning from './document-capture-warning';
 import DocumentCaptureReviewIssues from './document-capture-review-issues';
 
+// @ts-ignore
+type JSONObject = Record<string, string | number | boolean | null | JSONObject>;
```

Comment:
Using `// @ts-ignore` here masks the fact that the type is invalid, so will effectively be treated as an `any`.

![image](https://github.com/18F/identity-idp/assets/1779930/b98ba7d6-e651-49b6-82f4-1cb7124b6067)

We should avoid `@ts-ignore`, either by making this a valid type.

Alternatively, at points in the past I'd contemplated pulling in a TypeScript utility package like `type-fest`, which has various JSON types we could use here?

https://github.com/sindresorhus/type-fest#json

Reply:
Revert back to string.